orderer/common/cluster/util.go

```diff
@@ -73,7 +74,7 @@ func (mp MemberMapping) ByID(ID uint64) *Stub {
 // LookupByClientCert retrieves a Stub with the given client certificate
 func (mp MemberMapping) LookupByClientCert(cert []byte) *Stub {
 	for _, stub := range mp {
-		if bytes.Equal(stub.ClientTLSCert, cert) {
+		if crypto.CertificatesWithSamePublicKey(stub.ClientTLSCert, cert) == nil {
```

Comment:
Not an expert in this area of the code, but I'm assuming this ends up cached to prevent constantly iterating over and parsing the certs, looking for a match?